pavement.py

```diff
@@ -28,7 +28,12 @@
 #
 # =================================================================
 
-from ConfigParser import SafeConfigParser
+from __future__ import (absolute_import, division, print_function)
+
+try:
```

Comment:
Maybe the higher-level `from six.moves import configparser` will work? It works fine for me on 2.7 and 3.4, so maybe it is Paver's doing.